styles.css

```diff
@@ -0,0 +1,989 @@
+}
+
+.bottom-editors-header {
+  font-weight: bold;
+  font-size: 14px;
+  color: black;
+  text-decoration: none;
+  cursor: text;
+  margin: 20px 0;
+}
+
+.bottom-cards {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  margin-bottom: 20px;
+}
+
+.bottom-editors-card {
+  width: 32%;
+  display: flex;
+  flex-direction: column;
+}
+
+.bottom-editors-card img {
+  height: 200px;
+  width: 100%;
+}
+
+.bottom-editors-card p {
+  color: rgb(153, 150, 150);
+  font-size: 12px;
+  line-height: 20px;
+}
+
+.right-positioned {
+  align-self: flex-end;
+  color: rgb(153, 150, 150);
+  font-size: 10px;
+  line-height: 20px;
+}
+
+.bottom-editors-card a {
+  color: black;
+  text-decoration: none;
+  line-height: 20px;
+  font-size: 16px;
+}
+
+/* Styling the bottom links */
+
+.article-bottom {
+  border-top: 5px double rgb(226, 222, 222);
+  width: 80%;
+  margin: auto;
+  border-bottom: 1px solid rgb(214, 208, 208);
+}
+
+.bottom-header {
+  margin-top: 10px;
+  margin-bottom: 25px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.bottom-header-img {
+  height: 30px;
+}
+
+.bottom-links {
+  display: flex;
+  width: 100%;
+  margin-bottom: 10px;
+}
+
+.home-page {
+  display: flex;
+}
+
+.column {
+  width: calc(100% / 6);
+  font-family: nyt-franklin, helvetica, arial, sans-serif;
+  font-size: 12px;
+}
+
+.column h4 {
+  margin-bottom: 15px;
+  font-weight: bold;
+}
+
+.column li {
+  margin-bottom: 10px;
+}
+
+.article-bottom a {
+  height: 30px;
+}
+
+.column a {
+  color: black;
+  text-decoration: none;
+}
+
+.sbscribe-column {
+  border-left: 1px solid rgb(214, 208, 208);
+  padding-left: 15px;
+}
+
+.bold-links {
+  font-weight: bold;
+  font-size: 15px;
+  margin-bottom: 20px;
+}
+
+.smaller-size-links {
+  font-size: 10px;
+}
+
+.ssl-1 {
+  margin: 10px 0;
+  border-bottom: 1px solid rgb(214, 208, 208);
+}
+
+/* Styling the footer */
+
+footer {
+  width: 65%;
+  font-size: 12px;
+  display: flex;
+  justify-content: space-between;
+  margin: auto;
+  padding-top: 15px;
+  padding-bottom: 20px;
+}
+
+.footer-links {
+  color: rgb(99, 97, 97);
+  text-decoration: none;
+}
+
+/* Sytling hover actions for the whole page */
+
+.left-nav-section a:hover {
+  text-decoration: underline;
+}
+
+.right-nav-section button:hover {
+  background-color: #326891;
+}
+
+.figcaption-author a:hover {
+  border-bottom: unset;
+}
+
+.signup-btn:hover {
+  background-color: black;
+  color: white;
+}
+
+.pick a:hover {
+  text-decoration: underline;
+}
+
+.article-end a:hover {
+  text-decoration: none;
+}
+
+.bottom-btn:hover {
+  background-color: #326891;
+}
+
+.card-btn:hover {
+  background-color: black;
+  color: white;
+}
+
+.card a:hover {
+  text-decoration: underline;
+}
+
+.card:hover {
+  box-shadow: 0 2px 5px 2px #c9c8c8;
+}
+
+.most-pop-link a:hover {
+  color: rgb(85, 83, 83);
+}
+
+.more-a:hover {
+  text-decoration: underline;
+}
+
+.more-section-card a:hover {
+  color: rgb(85, 83, 83);
+}
```

Comment:
the ` color: rgb(85, 83, 83)` used in different class. Use DRY principle.

e.g. - 
```
.more-section-card a:hover,
.bottom-editors-card a:hover,
.most-pop-link a:hover
{
  color: rgb(85, 83, 83);
}
```